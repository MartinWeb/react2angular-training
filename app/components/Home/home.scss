body {
    margin: 0;
    background-color: honeydew;
    font-family: Georgia;

    #home {
        width: 80%;
        margin: auto;
    }
}