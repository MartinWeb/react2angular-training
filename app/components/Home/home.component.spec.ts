import { HomeComponent } from "./home.component";

describe('HomeComponent', () => {
    test('should have valid snapshot', () => {
        expect(HomeComponent.template).toMatchSnapshot();
    });
});