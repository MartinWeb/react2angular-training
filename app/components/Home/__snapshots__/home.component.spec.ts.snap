// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeComponent should have valid snapshot 1`] = `
"<header-component></header-component>

<div id="home">
    <alert-component title="Chuck Norris welcomes you !" type="info"></alert-component>
    <warning-component></warning-component>

    <choice-component></choice-component>
</div>

<footer-component></footer-component>"
`;
